docs/studios/unity/unity-sdk/creating-bots/javascript/agent-builder.mdx

```diff
@@ -0,0 +1,240 @@
+---
+sidebar_label: 'Agent Builder'
+---
+
+import BehaviorTree from '../../../../../common/agent-builder/_behavior-trees.md'
+import LoadingYourTree from '../../../../../common/agent-builder/_loading-your-tree.mdx'
+import AddingNodes from '../../../../../common/agent-builder/_adding-nodes.mdx'
+import SavingYourTree from '../../../../../common/agent-builder/_saving-your-tree.mdx'
+
+# Creating AIs Using the Agent Builder
+
+Regression Games' Agent Builder is an easy-to-use visual editor for constructing AI behavior trees.
+Our editor utilizes large language models like [OpenAI's GPT-4](https://openai.com/blog/chatgpt) to write code for you.
+This makes the Agent Builder an excellent tool for writing bots as you familiarize yourself with your project's SDK integration.
+
+:::note
+This tool is early in development - it is prone to change and may contain bugs.
+Please report issues and feature requests for the Agent Builder [here](https://regression-games.sleekplan.app/feedback).
```

Comment:
One thing I've learned from writing docs is to avoid writing content you **know** will change over time, if you *can* avoid it. 

Also "here" links are less accessible, as screen readers don't do a great job of describing them. It's better to give the link text that is specific to where the link goes, rather than relying on previous context in the sentence or paragraph.
 
So maybe rephrase this as:

```suggestion
We're constantly working to improve Agent Builder. You can report issues and submit feature requests on our [Feedback Site](https://regression-games.sleekplan.app/feedback).
```

Reply:
Updated!